CONTRIBUTING.md

```diff
@@ -63,7 +111,7 @@ We have very precise rules over how our git commit messages can be formatted. Th
 Each commit message consists of a **header**, a **body** and a **footer**. The header has a special
 format that includes a **type**, a **scope** and a **subject**:
 
-```
+```text
```

Comment:
nit. Markdownlint (I used this plugin) check this

https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md040.md